docs/source/release_notes/release-notes.rst

```diff
@@ -63,7 +63,11 @@ develop
            This has empirically shown to be a better batch size because it puts less stress on the underlying KVS.
            For a full list of tunable sweep parameters and default settings, see :ref:`sweep tunable options <sweep_tunable_parameters>`.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1763>`__)
-       
+
+    *    - |improved|
```

Comment:
Did not add any new documents as did not find it necessary, lock requests taking more than 100ms represents a problem and I don't think there should be too many occurrences. If there need be, we can have the docs to configure a separate appender!